src/api/game/get/queries.ts

```diff
@@ -0,0 +1,10 @@
+import { getRequest } from "../../request";
+import { gamesByUserIdUrl } from "../get/urls";
```

Comment:
I would still prefer 
```suggestion
import { gamesByUserId } from "../get/urls";
```

because you are importing it from `../get/urls` and invoking parameter is `url:` so, it's understandable from the context that it's a url, there's no need to enhance a name with the `url` in the end of it.